src/index.js

```diff
@@ -26,17 +46,67 @@ function removeProduct(event) {
   const target = event.currentTarget;
   console.log('The target in remove is:', target);
   //... your code goes here
-}
+    let remov = target.parentNode.parentNode;
+    remov.remove()
```

Comment:
Here, you could've passed the calculateAll function to calculate the prices of all remaining products after removing one!